albumentations/augmentations/transforms.py

```diff
@@ -3320,3 +3322,91 @@ def get_transform_init_args_names(self):
+        alias_blur ((float, float) or float): range for alias_blur of defocusing (sigma of gaussian blur).
+            If limit is a single float, the range will be (0, limit). Default: (0.1, 0.5).
+        p (float): probability of applying the transform. Default: 0.5.
+
+    Targets:
+        image
+
+    Image types:
+        Any
+    """
+
+    def __init__(self, radius=(3, 10), alias_blur=(0.1, 0.5), always_apply=False, p=0.5):
+        super(Defocus, self).__init__(always_apply, p)
+        self.radius = to_tuple(radius, low=1)
+        self.alias_blur = to_tuple(alias_blur, low=0)
+
+        if self.radius[0] < 0:
+            raise ValueError("Parameter radius must be positive")
+
+        if self.alias_blur[0] <= 0:
+            raise ValueError("Parameter alias_blur must be non-negative")
+
+    def apply(self, img, radius=3, alias_blur=0.5, **params):
+        return F.defocus(img, radius, alias_blur)
+
+    def get_params(self):
+        return {
+            "radius": random.choice(np.arange(self.radius[0], self.radius[1] + 1)),
+            "alias_blur": random.uniform(self.alias_blur[0], self.alias_blur[1]),
+        }
+
+    def get_transform_init_args_names(self):
+        return ("radius", "alias_blur")
+
+
+class ZoomBlur(ImageOnlyTransform):
+    """
+    Apply zoom blur transform. See arXiv:1903.12261 [cs.LG].
+
+    Args:
+        max_factor ((float, float) or float): range for max factor for blurring.
+            If max_factor is a single float, the range will be (1, limit). Default: (1, 1.31).
+            All max_factor values should be larger than 1.
+        step_factor ((float, float) or float): If single float will be used as step parameter for np.arange.
+            If tuple of float step_factor will be in range `[step_factor[0], step_factor[1])`. Default: (0.01, 0.03).
+            All step_factor values should be positive.
+        p (float): probability of applying the transform. Default: 0.5.
+
+    Targets:
+        image
+
+    Image types:
+        Any
+    """
+
+    def __init__(self, max_factor=1.31, step_factor=(0.01, 0.03), always_apply=False, p=0.5):
+        super().__init__(always_apply, p)
+        self.max_factors = to_tuple(max_factor, low=1.0)
+        self.step_factors = to_tuple(step_factor, step_factor)
+
+        if self.max_factors[0] < 1:
+            raise ValueError("Max factor must be larger or equal 1")
+        if self.step_factors[0] <= 0:
+            raise ValueError("Step factor must be positive")
+
+    def apply(self, img, zoom_factors=np.arange(1, 1.11, 0.01), **params):
+        return F.zoom_blur(img, zoom_factors)
+
+    def get_params(self):
+        max_factor = random.uniform(self.max_factors[0], self.max_factors[1])
+        step_factor = (
+            self.step_factors[0]
+            if self.step_factors[0] == self.step_factors[1]
+            else random.uniform(self.step_factors[0], self.step_factors[1])
+        )
```

Comment:
Do not need to check that values are equal.
`random.uniform(self.step_factors[0], self.step_factors[1])` works well, when `self.step_factors[0] == self.step_factors[1]`

Reply:
Oh, no reason, will fix.